lib/core/src/Cardano/Wallet.hs

```diff
@@ -110,6 +110,14 @@ module Cardano.Wallet
 
     -- ** Root Key
     , withRootKey
+
+    -- ** Stake Pools
+    , createCert
+    , signCert
+    , submitCert
+    , ErrCreateCert (..)
```

Comment:
Same point as I made here https://github.com/input-output-hk/cardano-wallet/pull/1059#discussion_r349318171: 

Because there are multiple kinds of certificates in jormungandr I find these names a bit ambiguous.

What do you think about:
- `createDelegationCert`
- `signDelegationCert`
- `submitDelegationCert`
- `ErrCreateDelegationCert`

Reply:
sound argument, I will address it in the signCert (or signDelegationCert) PR soon

Reply:
I agree that naming could be clearer but in the context of the wallet, the term certificate is non-ambiguous. Jörmungandr is out of the question at this level (as well as in the API). 

Yet, it wouldn't hurt to add `Delegation` to make things clearer indeed.